includes/REST_API/Stories_Controller.php

```diff
@@ -265,6 +265,57 @@ public function filter_posts_clauses( $clauses, $query ): array {
 		return $clauses;
 	}
 
+	/**
+	 * Prime post caches for attachments and parents.
+	 *
+	 * @since 1.21.0
+	 *
+	 * @param WP_Post[] $posts Array of post objects.
+	 * @return mixed Array of posts.
+	 */
+	public function prime_post_caches( $posts ) {
+		$post_ids = $this->get_attached_post_ids( $posts );
+		if ( ! empty( $post_ids ) ) {
+			_prime_post_caches( $post_ids );
+		}
+		$user_ids = $this->get_attached_user_ids( $posts );
+		if ( ! empty( $user_ids ) ) {
+			cache_users( $user_ids );
+		}
+
+		return $posts;
+	}
+
+	/**
+	 * Get an array of attached post objects.
+	 *
+	 * @since 1.21.0
```

Comment:
Nit: the next version is 1.22.0, since 1.21.0 has been released already.

Reply:
```suggestion
	 * @since 1.22.0
```